stingray/crossspectrum.py

```diff
@@ -9,6 +9,7 @@
 
 import stingray.lightcurve as lightcurve
 import stingray.utils as utils
+from stingray._exceptions import StingrayError
```

Comment:
I think it should be fine if we name the module without the underscore, like [NetworkX](https://networkx.github.io/documentation/networkx-1.10/reference/exceptions.html)